bucket/ptags.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "0.3.0",
+    "description": "A parallel universal-ctags wrapper for git repository",
+    "license": "MIT",
+    "url": "https://github.com/dalance/ptags/releases/download/v0.3.0/ptags-v0.3.0-x86_64-win.zip",
+    "depends": "ctags",
+    "bin": "ptags.exe",
+    "homepage": "https://github.com/dalance/ptags",
```

Comment:
Move before license

Reply:
url, depends, bin, and homepage or just homepage?

Reply:
Comment is for L8. What exactly make you feel like it was for all lines above?